### PR TITLE
Jpk/things and stuff

### PR DESCRIPTION
**Pull request by jpknoll**
I need to fix a lot of the specs still.

### CODE REVIEW

**Comment by jSylvestre** [comment]
The payment choice on the create page appears to be messed up

File: Anlab.Mvc/Client/components/PaymentSelection.tsx [hunk 1]
```diff
@@ -1,115 +1,126 @@
...
+            accountName: null,
             error: "",
-            accountName: null
         };
+    }
 
+    public render() {
+        const activeDiv = "anlab_form_style col-5 active-border active-text active-bg";
+        const inactiveDiv = "anlab_form_style col-5";
+        const isUcClient = this.props.payment.clientType !== "uc";
```
Comment (jSylvestre) [inline Anlab.Mvc/Client/components/PaymentSelection.tsx]: @jpknoll Backwards?

Comment (jpknoll) [reply Anlab.Mvc/Client/components/PaymentSelection.tsx]: Yup! Fix incoming.

File: Anlab.Mvc/Client/components/PaymentSelection.tsx [hunk 2]
```diff
@@ -1,115 +1,127 @@
...
-                .then(response => response.json())
-                .then(accountName => this.setState({ accountName }))
-                .catch(error => {
-                    this.setState({ accountName: null, error: "The account you entered could not be found" });
-                });
-        }
-        else
-        {
-            this.setState({ accountName: null });
+    private _lookupAccount = () => {
```
Comment (jSylvestre) [inline Anlab.Mvc/Client/components/PaymentSelection.tsx]: @jpknoll There is a problem here clearing out the error message when it isn't a UC Chart (3 or L is UC)

**Pull request merged**